### PR TITLE
Create DMWands/Patch_Weapons.xml @ Patches

### PR DESCRIPTION
**Pull request by d4egon**
Preview

## Additions

Patch 1.0
- Compatibility patch 

## Changes

Removes and adds the existing weapons and magic bolts (Bullets) with PatchOperation to achieve compatibility with CE

## References

- Closes #[773]

## Reasoning

CE is awesome? and I'd like my mod to be as usable as possible

## Alternatives

Not relevant

## Testing

Check tests you have performed:
- [x] Compiles without warnings
- [x] Game runs without errors
- [x] (For compatibility patches) ...with and without patched mod loaded
- [x] Playtested a colony (5hrs)

### CODE REVIEW

File: Patches/DMWands/Patch_Weapons.xml [hunk 1]
```diff
@@ -0,0 +1,305 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+
+	<Operation Class="PatchOperationFindMod">
+    <mods>
+		<li>D4egons Magic Wands</li>
```
Comment (N7Huntsman) [inline Patches/DMWands/Patch_Weapons.xml:6]: Name is incorrect, should be `Magic Wand` to match the mod name in About.xml

File: Patches/DMWands/Patch_Weapons.xml [hunk 2]
```diff
@@ -0,0 +1,305 @@
...
+			</li>
+			
+			<li Class="PatchOperationRemove">
+				<xpath>/Defs/ThingDef[defName="DM_MagicBolt"]</xpath>
+			</li>
+
+			<li Class="PatchOperationAdd">
+				<xpath>/Defs</xpath>
+				<value>
+					<ThingDef Class="CombatExtended.AmmoDef" ParentName="AmmoBase">
```
Comment (N7Huntsman) [inline Patches/DMWands/Patch_Weapons.xml:44]: For all these wand projectiles, remove `Class="CombatExtended.AmmoDef"` and set the Parent to `BaseBullet`. AmmoBase is for the ammo item, not the projectile.

File: Patches/DMWands/Patch_Weapons.xml [hunk 3]
```diff
@@ -0,0 +1,305 @@
...
+							<texPath>Things/Item/Equipment/WeaponRanged/DM_MagicBolts/DM_PoisonBolt</texPath>
+							<graphicClass>Graphic_Single</graphicClass>
+							<shaderType>TransparentPostLight</shaderType>
+							<color>(100,255,100)</color>
+						</graphicData>
+						<projectile Class="CombatExtended.ProjectilePropertiesCE">
+							<damageDef>DM_Poison</damageDef>
+							<armorPenetrationBlunt>20</armorPenetrationBlunt>
+							<armorPenetrationSharp>10</armorPenetrationSharp>
+							<speed>5</speed>
```
Comment (N7Huntsman) [inline Patches/DMWands/Patch_Weapons.xml:129]: Might increase this projectile speed somewhat, to 15 - 20.

**Review by N7Huntsman — CHANGES_REQUESTED** [review]
Some changes to be made.

**Review by d4egon — COMMENTED** [review]
Changed the things requested

**Review by d4egon — COMMENTED** [review]
checked the viewed button....

**Review by N7Huntsman — APPROVED** [review]

**Pull request merged**